assets/scripts/components/global-modals.js

```diff
@@ -17,7 +17,7 @@ const doOnLoad = () => {
         }
 
         if (lang === 'fr' || lang === 'ja' || lang === 'ko') {
```

Comment:
Huh, interesting that we limit to these languages considering we have others, like Portuguese and Spanish. I wonder if that's intended or not. Maybe the documentation page supports only these three?

Reply:
Yeah it seems that is the case. There is a drop down on the bottom of the documentation page with these languages.